src/main/java/zmaster587/advancedRocketry/asm/ClassTransformer.java

```diff
@@ -746,28 +750,17 @@ public ClassTransformer() {
 						if(numALoadsInARow == 0)
 							firstALoadIndex = i;
 						numALoadsInARow++;
-						if(numALoadsInARow == 2) {
-							pos = ain;
+						if(numALoadsInARow == 4) {
 							break;
 						}
 					}
 					else
 						numALoadsInARow = 0;
 				}
 				int i = firstALoadIndex;
-				while((ain = onUpdate.instructions.get(--i)).getOpcode() != Opcodes.PUTFIELD);
```

Comment:
If memory serves, this bit was to remove the part where vanilla gravity is applied, we sure we want it removed?

Reply:
Yes, because the ASM no longer removes gravity application, as it was inconsistent between things.